dataprofiler/profilers/unstructured_text_profile.py

```diff
@@ -100,12 +100,33 @@ def __init__(self, name, options=None):
         if options and options.stop_words is not None:
             self._stop_words = options.stop_words
 
+        self._top_k_chars = None
+        if options:
+            self._top_k_chars = options.top_k_chars
+
+        self._top_k_words = None
+        if options:
+            self._top_k_words = options.top_k_words
```

Comment:
Van we simply these two if statements?

Reply:
Do you have a suggestion? @gme5078 Not sure we can simplify it further.

Reply:
self._top_k_chars = None
self._top_k_words = None

        if options:
            self._top_k_chars = options.top_k_chars
            self._top_k_words = options.top_k_words